azavea/franklin
# Decode requested extensions on serialization

## Overview

This PR decodes user-requested extensions on the collection items list endpoint. It doesn't do so in search because:

1. this accomplishes proof of concept
2. there would be two ways to express filtering for items containing an extension (`extensions=...` or using the, but in only one case would it be obvious / super detectable that a user wanted extensions

### Checklist

- [x] New tests have been added or existing tests have been modified (and they don't even pass right now :tada:)

### Notes

need to pass tests

### Testing Instructions

- send some `extensions==label,layer` and other combinations parameters
- note that you get validation results back for items and their links
- it's hard to test outside of labels right now because we don't have catalogs available for every extension that we can import, but new tests verify consistency with multiple extensions

Closes #147

Closes #30
